docs/design/3_1_2_platform_management.tex

```diff
@@ -35,18 +35,19 @@ \subsubsection{Data Flow}
 All operations are managed by the ARM Cortex A72 chip running a headless version of the Raspbian operating system. 
 
 \subsubsection{Use of Multiple Computing Boards}
-The presence of on-board WiFi/camera interfaces is the primary driving factor towards adopting a dual-board (Platform Management and Programmable) solution. While a single-board solution is possible, the development effort and cost of the computing platform would be considerably higher. Most programmable logic boards (including our chosen board, the Zedboard) require external expansion devices to interface with peripherals such as WiFi/cameras. These expansion devices, which connect to the programmable logic board via a serial connection, have poorly documented drivers and are of considerable expense -- ranging from US\$25 for a basic WiFi module\cite{digiwifi} to US\$200 for a compatible camera interface card\cite{digipmod}. External expansion devices also require considerable glue logic resources on the programmable logic board, potentially risking non-compliance with constraint \textbf{C.EX.2}.
+The presence of on-board WiFi/camera interfaces is the primary driving factor towards adopting a dual-board (Platform Management and Programmable) solution. While a single-board solution is possible, the development effort and cost of the computing platform would be considerably higher. Most programmable logic boards (including our chosen board, the Zedboard) require external expansion devices to interface with peripherals such as WiFi/cameras. These expansion devices, which connect to the programmable logic board via a serial connection, have poorly documented drivers and are of considerable expense -- ranging from US\$25 for a basic WiFi module\cite{digiwifi} to US\$200 for a compatible camera interface card\cite{digipmod}. External expansion devices also require considerable glue logic on the programmable logic board, potentially risking non-compliance with constraint \textbf{C.EX.2}.
```

Comment:
```suggestion
The presence of on-board Wi-Fi and camera interfaces is the primary driving factor towards adopting a dual-board (PMB and PLB) solution. While a uni-board solution is possible, the development effort and cost of the computing platform would be considerably higher. Most programmable logic boards (including our chosen board, the Zedboard) require external expansion devices to interface with peripherals such as WiFi/cameras. These expansion devices, which connect to the programmable logic board via a serial connection, have poorly documented drivers and are of considerable expense -- ranging from US\$25 for a basic WiFi module\cite{digiwifi} to US\$200 for a compatible camera interface card\cite{digipmod}. External expansion devices also require considerable glue logic on the programmable logic board, potentially risking non-compliance with constraint \textbf{C.EX.2}.
```

Reply:
Additional reasoning:

- Everything fitted onto a single board would have longer turn-around time for design iterations
- Uni-board is harder to integrate with the drone as it is less modular --- reduces mechanical freedom
- Uni-board goes against the requirement/objective of expandability